src/model/scripts/spm_training/README.md

```diff
@@ -1,6 +1,28 @@
-### Argument
+# OpenThaiGPT tokenizer training pipeline
 
-- output_path
-- vocab_size
-- data_path
-- data_type
\ No newline at end of file
+## Description
+
+The OpenThaiGPT tokenizer training pipeline is a pipeline for training tokenizer specially for Thai language.
+
+## how to train
+
+1. you need to config argument first [configuration_example/spm/training_v1.yaml](../../configuration_example/spm/training_v1.yaml)
+
+   - output_path (str): The path and prefix to use when saving the trained tokenizer.
+   - vocab_size (int): The size of the vocabulary to use when training the tokenizer. Defaults to the number of available CPU cores.
+   - is_slurm (bool): Whether the code is running on a Slurm cluster. Defaults to False.
```

Comment:
What will happened if we set is_slurm = True, False?

Reply:
It is streaming of dataset I will change is_slurm to streaming